spec/classes/collectd_plugin_iptables_spec.rb

```diff
@@ -26,6 +26,20 @@
         end
       end
 
+      context ':ensure => present and :chains6 => { \'nat\' => \'In6_SSH\' }' do
```

Comment:
Can you also use ```filter``` instead of ```nat```? I know it doesn't matter that much, but it feels wrong :)